### PR TITLE
support of uploading native symbols and source code

### PR DESCRIPTION
**Pull request by marandaneto**
fix docs for https://github.com/getsentry/sentry-java/pull/815

### CODE REVIEW

File: src/collections/_documentation/platforms/android/index.md [hunk 1]
```diff
@@ -231,6 +231,18 @@ sentry {
...
     // upload the mapping files with sentry-cli when you do a release.
     autoUpload true
+
+    // Disables or enables the automatic configuration of Native Symbols
+    // for Sentry. This executes sentry-cli automatically so
+    // you don't need to do it manually.
+    // Default is disabled.
+    uploadNativeSymbols false
+
+    // Includes or not the source code of native code for Sentry.
```
Comment (MimiDumpling) [inline src/collections/_documentation/platforms/android/index.md]:
```suggestion
    // Does or doesn't include the source code of native code for Sentry.
```

File: src/collections/_documentation/platforms/android/index.md [hunk 2]
```diff
@@ -478,15 +490,15 @@ dependencies {
 }
 ```
 
-If you want to use the SDK with the NDK but you still want to support the devices on the API level lower than 21 you can update your manifest as follows:
+If you want to use the SDK with the NDK but you still want to support the devices on the API level lower than 16 you can update your manifest as follows:
```
Comment (MimiDumpling) [inline src/collections/_documentation/platforms/android/index.md]:
```suggestion
If you want to use the SDK with the NDK, but you still want to support the devices on the API level lower than 16, you can update your manifest as follows:
```

File: src/collections/_documentation/platforms/android/index.md [hunk 3]
```diff
@@ -478,15 +490,15 @@ dependencies {
...
 
 ```xml
     <!-- Merging strategy for the imported manifests -->
       <uses-sdk
             tools:overrideLibrary="io.sentry.android" />
 ```
 
-With these changes, the NDK integration is going to be used only on the devices with the API level ≥ 21 and the rest of the devices with API level ≥14, but ≤21 will use just the SDK.
+With these changes, the NDK integration is going to be used only on the devices with the API level ≥ 16 and the rest of the devices with API level ≥14, but ≤16 will use just the SDK.
```
Comment (MimiDumpling) [inline src/collections/_documentation/platforms/android/index.md]: I tried making this a bit more digestible. Hoping it's still valid.
```suggestion
With these changes, the NDK integration will be used only on:
- the devices with the API level ≥ 16  
- the rest of the devices with API level ≥ 14

Any device ≤ 16 will use the SDK.
```

Comment (marandaneto) [reply src/collections/_documentation/platforms/android/index.md]: we (@fdrabek too) simplified a little bit, hopefully, it's good to go :)

**Pull request merged**